### ISSUE TITLE
Support cross-namespace operation

### ISSUE THREAD

**Issue by jaredpetersen** [arangodb/kube-arangodb#401]
It would be great if the ArangoDB operator was able to perform its operations across multiple namespaces rather than just the one it is currently deployed to. This would allow system administrators to deploy an operator once in a single "arangodb-operator" namespace instead of having to deploy it for each namespace that requires an ArangoDB installation.

**Comment by fico-jessecarroll** [comment]
I agree with this.  Having the operator watch for deployments across the cluster would add more value.

**Comment by uditha-atukorala** [comment]
IMHO, this is an essential feature. As it is right now, I can't really use the manifests directly [as described in the manual](https://www.arangodb.com/docs/stable/deployment-kubernetes-usage.html#installation-with-kubectl) since it installs everything in `default` namespace. And I have to deploy ArangoDB in the default namespace as well.  This is probably ok for simple clusters but for complicated clusters with many namespaces (which is the case for us), this doesn't work. 

I have to manually manage ArangoDB operator manifests and try to figure out what changes are made when having to upgrade. This is very time consuming.

**Comment by cristichiru** [comment]
Why not use 1 namespace for the operator with multiple arangodb clusters?

One benefit of deploying each operator in its own namespace would be having different operator versions in the same cluster, each delimited by a namespace. I encounter this situation a lot, as I slowly upgrade various components in a cluster, and it is not always smooth.

Using the helm charts, you can deploy in a different namespace.

Bcs I do not really like helm, I have built an [ansible role](https://github.com/Revomatico/ansible-arangodb-operator) that automates operator deployment. using kustomize to change the namespace, and even prefix for all the object names and references, so you could have multiple operators within the same namespace.

**Comment by kmrhemant916** [comment]
Any update on this ?

**Comment by rob-64** [comment]
Is this possible yet? Seems like its a simple configuration change?

	rw := k8sutil.NewResourceWatcher(
		o.Client.Arango().DatabaseV1().RESTClient(),
		deploymentType.ArangoDeploymentResourcePlural,
		**o.Config.Namespace**,
